govcd/sample_govcd_test_config.yaml

```diff
@@ -60,6 +60,15 @@ vcd:
     internalIp: 192.168.1.10
     # An external Network name
     externalNetwork: myexternalnet
+    #
+    # Independent disk parameters for testing
+    disk:
+      #
+      # Disk size (bytes) for create disk, skip disk tests if it is less than 0
```

Comment:
I think better way use Mega bytes

Reply:
The min unit of disk size is byte in disk functions. If it change to MB, the test will not be able to create disk in bytes for testing e.g. Create 500.5MB Disk.

Reply:
Ok. If VCD allows bytes, let's leave it :+1: In terraform provider we will need to be more abstract, as now one need 1bytes disks, I guess.

Reply:
Good comment above. In general:
- For go-vcloud-director - we're aiming to be a thin layer on vCD API and _not_ be too smart about it;
- For Terraform vCD provider - we're aiming to make it easy and practical for users to use (e.g. using MBs instead of bytes).